[ BUG ] ImportError when running AWS Lambda

**Describe the bug**
The `util/create-lambda-layer.sh` script pip installs `crowdstrike-falconpy` with no other version restrictions, but since the release of `urllib3 >= 2.0`, this produces a broken layer since the Python 3.8 Lambda environment provides `OpenSSL 1.0.2-fips` instead of the minimum requirement from urllib3 of `OpenSSL 1.1.1`.

As a result, if you deploy the lambda from any of the AWS integration guides (e.g. - I was working with the [Security Hub setup guide](https://github.com/CrowdStrike/Cloud-AWS/tree/main/Security-Hub)), the lambda will raise the same error every time it runs:

```
[ERROR] Runtime.ImportModuleError: Unable to import module 'main': urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'OpenSSL 1.0.2k-fips  26 Jan 2017'. See: https://github.com/urllib3/urllib3/issues/2168
```

**To Reproduce**
To _truly_ reproduce, you just have to run the script, create a lambda from it, and try importing `urllib3`.  To reproduce locally (a bit more expedient):

```bash
# Clear out any previous experiments
$ cd util && rm -rf python/ falconpy-layer.zip

# Produce the lambda ZIP file
$ ./create-lambda-layer.sh

# Extract it (making sure to blow away any previous experiment
$ unzip falconpy-layer.zip

# Mount it into a lambda/python:3.8 container and see that it fails
$ docker run -it --rm --entrypoint '' --rm \
    -v $(pwd)/python:/app --workdir /app -e PYTHONPATH=. \
    public.ecr.aws/lambda/python:3.8 python -c 'import urllib3'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/app/urllib3/__init__.py", line 41, in <module>
    raise ImportError(
ImportError: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'OpenSSL 1.0.2k-fips  26 Jan 2017'. See: https://github.com/urllib3/urllib3/issues/2168
```

**Expected behavior**
The import of `urllib3` should succeed (thus indicating this issue will be fixed when deployed as an actual Lambda)

**Environment (please complete the following information):**
 - OS: 
   - Local: MacOS 13.4, GNU Bash 5.2.15
   - Remote: AWS Lambda - Python 3.7 (as per the cloudformation templates here: https://github.com/CrowdStrike/Cloud-AWS/blob/main/Security-Hub/cloudformation/security-hub-integration.yaml#L188)
 - Python: 3.7 and 3.8
 - FalconPy: `main` branch at commit 5883efcf2eea3243ef0bec4049a1685a87f4f7cc

**Additional context**
As it stands, the guides for setting up some of the AWS Crowdstrike integrations are broken until this is fixed.

--- a/AUTHORS.md
+++ b/AUTHORS.md
@@ -87,6 +87,7 @@ This has been a critical element in the development of the FalconPy project.
 + Chris, `@chrisbvt`
 + Alex, `@khyberspache`
 + Phil Massyn, `@massyn`
++ Russell Snyder, `@rusnyder`
 
 
 ## Sponsors

--- a/util/create-lambda-layer.sh
+++ b/util/create-lambda-layer.sh
@@ -7,11 +7,14 @@
 # Assumes the docker service is running.
 #
 # Currently targets Python 3.8
-echo "crowdstrike-falconpy" > lambda-requirements.txt
+cat - << EOF > lambda-requirements.txt
+crowdstrike-falconpy
+urllib3 < 2.0
+EOF
 # Remove any old copies
 rm falconpy-layer.zip >/dev/null 2>&1
 # Run the lambci image and install the requirements
-docker run --rm -v $(pwd):/foo -w /foo lambci/lambda:build-python3.8 \
+docker run --rm --entrypoint '' -v $(pwd):/foo -w /foo public.ecr.aws/lambda/python:3.8 \
 pip install -r lambda-requirements.txt -t python
 # Create the layer archive
 zip -r falconpy-layer.zip python

